### DISCUSSION TITLE
92D03309-D3D4-B207-02F1-E63980F27A14

### DISCUSSION THREAD

**Discussion by giscus** [celestiamobile/comments#77] [General]
# 92D03309-D3D4-B207-02F1-E63980F27A14

Mu Arae, also known as Cervantes, is a Sun-like star about 50.9 light-years away in the constellation Ara. It has three Jovians and one planet that is either a superterran or a hot Neptunian (this addon depicts that planet as a superterran). Although there are no known exomoons to exist on this planetary system, there are three fictional exomoons orbiting each gas giant, one of which is a potentially habitable moon orbiting Mu Arae b (Quijote)

https://celestia.mobi/addon?item=92D03309-D3D4-B207-02F1-E63980F27A14&lang=en

**Comment by exocritic5457** [comment]
The description says there are 3 (fictional) moons around each gas giant, when it's actually 1 moon per planet each and the system has 3 moons in total because it has 3 gas giants, not any 9 moons like it may imply. Mu Arae e's moon is labeled "Mu Arae b-I' despite being e's. Overall this addon is outdated in that c is rocky despite being well-known to likely be a hot Neptune instead.

**Reply by DaveBowman2001** [reply]
The description should now clarify that there's one (fictional) exomoon orbiting each gas giant in the system, and the name of Mu Arae e's moon should now be fixed as well (alongside receiving a fictional tag on each one of them)
